v2/internal/testcommon/kube_per_test_context.go

```diff
@@ -566,10 +580,20 @@ func (tc *KubePerTestContext) ExpectResourceDoesNotExist(key types.NamespacedNam
 }
 
 // LogSection creates a distinctive header in the log to aid scanning
-func (tc *KubePerTestContext) LogSection(section string) {
-	line := strings.Repeat("=", 50)
+func (tc *KubePerTestContext) LogSectionf(section string, args ...any) {
+	msg := fmt.Sprintf(section, args...)
+	line := strings.Repeat("=", len(msg))
+	tc.T.Log(line)
+	tc.T.Log(msg)
+	tc.T.Log(line)
+}
+
+// LogSection creates a distinctive header in the log to aid scanning
+func (tc *KubePerTestContext) LogSubsectionf(subsection string, args ...any) {
+	msg := fmt.Sprintf(subsection, args...)
+	line := strings.Repeat("-", len(msg))
 	tc.T.Log(line)
-	tc.T.Log(section)
+	tc.T.Log(msg)
 	tc.T.Log(line)
 }
```

Comment:
I'd suggest something like below here to re-use the code:
```
// LogSection creates a distinctive header in the log to aid scanning
func (tc *KubePerTestContext) LogSectionf(section string, args ...any) {
	tc.logf(subsection, "=", args)
}

// LogSection creates a distinctive header in the log to aid scanning
func (tc *KubePerTestContext) LogSubsectionf(subsection string, args ...any) {
	tc.logf(subsection, "-", args)
}

func (tc *KubePerTestContext) logf(s string, r string, args ...any) {
	msg := fmt.Sprintf(s, args...)
	line := strings.Repeat(r, len(msg))
	tc.T.Log(line)
	tc.T.Log(msg)
	tc.T.Log(line)
}
```

Reply:
Done.